spec/robots/accession/technical_metadata_spec.rb

```diff
@@ -88,18 +126,99 @@
         end
       end
 
-      context 'when the DRO has no files' do
-        let(:object) do
-          Cocina::Models::DRO.new(externalIdentifier: '123',
-                                  type: Cocina::Models::Vocab.object,
-                                  label: 'my repository object',
-                                  version: 1)
+      context 'when file has true for sdrPreserve' do
+        before do
+          stub_request(:post, 'https://dor-techmd-test.stanford.test/v1/technical-metadata')
```

Comment:
Not a blocker but seeing all of these stub requests in this spec I wonder if we should be creating an integration test as well.

Reply:
definitely!  I'll make a ticket for that.

Reply:
ticket existed - i expanded the description a bit.  https://github.com/sul-dlss/infrastructure-integration-test/issues/24